app/views/games/show.html.erb

```diff
@@ -1,102 +1,102 @@
 <head>
 <%= stylesheet_link_tag 'style' %>
- 
+
 <script type="text/javascript">
 var correctCards = 0;
 $( init );
- 
+
 function init() {
   // Hide the success message
   $('#successMessage').hide();
   $('#successMessage').css( {
-    left: '580px',
-    top: '250px',
-    width: 0,
-    height: 0
-  } );
- 
-  // Reset the game
-  correctCards = 0;
-  //$('#cardPile').html( '' );
-  $('#cardSlots').html( '' );
- 
-  // Create the pile of shuffled cards
-  var numbers = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-  numbers.sort( function() { return Math.random() - .5 } );
- 
-  for ( var i=0; i<<%= @cards.size %>; i++ ) {
-    $("#card"+i).click(function(event) {
+left: '580px',
```

Comment:
Some of that javascript isn't used and can be removed.

For now i think clarity trumps execution speed. I don't think we need to be
worried about how expensive this operation is because 1.there's no traffic
2. In the grand scheme of things it's not that expensive of an operation.

My main gripe is that the helper method you're specifying is deprecated,
and the method I was using is not.
On May 10, 2015 10:02 PM, "Michael Smith" <notifications@github.com> wrote:

> In app/views/games/show.html.erb
> <https://github.com/bcwik9/crushcard/pull/7#discussion_r30009893>:
>
> > -    width: 0,
> > -    height: 0
> > -  } );
> > -
> > -  // Reset the game
> > -  correctCards = 0;
> > -  //$('#cardPile').html( '' );
> > -  $('#cardSlots').html( '' );
> > -
> > -  // Create the pile of shuffled cards
> > -  var numbers = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
> > -  numbers.sort( function() { return Math.random() - .5 } );
> > -
> > -  for ( var i=0; i<<%= @cards.size %>; i++ ) {
> > -    $("#card"+i).click(function(event) {
> > +left: '580px',
>
> This whole file is fucked up.
> I've refactored this in my second pull request to put all the javascript
> in classes in app/assets/javascript/*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bcwik9/crushcard/pull/7/files#r30009893>.
>